test-utils/src/dev/clickhouse.rs

```diff
@@ -141,6 +106,153 @@ impl ClickHouseInstance {
         })
     }
 
+    /// Start a new replicated ClickHouse server on the given IPv6 port.
+    pub async fn new_replicated(
+        port: String,
+        tcp_port: String,
+        interserver_port: String,
+        name: String,
+        r_number: String,
+        config_path: PathBuf,
+    ) -> Result<Self, anyhow::Error> {
+        let data_dir = TempDir::new()
+            .context("failed to create tempdir for ClickHouse data")?;
+        let log_path = data_dir.path().join("clickhouse-server.log");
+        let err_log_path = data_dir.path().join("clickhouse-server.errlog");
+        let tmp_path = data_dir.path().join("tmp/");
+        let user_files_path = data_dir.path().join("user_files/");
+        let access_path = data_dir.path().join("access/");
+        let format_schemas_path = data_dir.path().join("format_schemas/");
+        let args = vec![
+            "server".to_string(),
+            "--config-file".to_string(),
+            format!("{}", config_path.display()),
+        ];
+
+        let child = tokio::process::Command::new("clickhouse")
+            .args(&args)
+            .stdin(Stdio::null())
+            .stdout(Stdio::null())
+            .stderr(Stdio::null())
+            .env("CLICKHOUSE_WATCHDOG_ENABLE", "0")
```

Comment:
We should probably do `.env_clear()` here to be safe.

Reply:
same